openpype/hosts/maya/plugins/publish/collect_arnold_scene_source.py

```diff
@@ -35,6 +35,11 @@ def process(self, instance):
         # camera.
         cameras = cmds.ls(type="camera", long=True)
         renderable = [c for c in cameras if cmds.getAttr("%s.renderable" % c)]
+        if not renderable:
+            raise ValueError(
+                "No renderable cameraes found, which is required for "
+                "publishing ASS."
+            )
```

Comment:
```suggestion
        if not renderable:
            raise ValueError(
                "No renderable cameras found, which is required for "
                "publishing ASS."
            )
```

Also, why is a camera required? Can't you export an ass file of just geometry?

Reply:
Additionally, this error should actually not be occurring in collect but during validation.

Reply:
Totally. If we're touching it, let's convert it into a validation with `PublishValidationError` so it's compatible with the new publisher right away.

Reply:
Good point that is not required.